test/lwc-dev-server/index.test.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+import { expect } from 'chai';
+import { Logger } from '@salesforce/core';
+import { LWCServer } from '@lwc/lwc-dev-server';
+import esmock from 'esmock';
+import * as devServer from '../../src/lwc-dev-server/index.js';
+// eslint-disable-next-line no-underscore-dangle
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+const logger = {
+  debug: () => {},
+  warn: () => {},
+  trace: () => {},
+  getLevel: () => 10,
+} as Logger;
+
+describe('lwc-dev-server', () => {
```

Comment:
Can we have a test which can use a sample sfdx project and start the lwc dev server and stop it successfully? Like a sanity end-2-end test.